src_i18n/041_get_positions.ts

```diff
@@ -23,12 +22,14 @@ async function main() {
 
   //LANG:JP 全てのトークンアカウントを取得
   //LANG:EN Get all token accounts
-  const token_accounts = (await ctx.connection.getTokenAccountsByOwner(ctx.wallet.publicKey, {programId: TOKEN_PROGRAM_ID})).value;
```

Comment:
Could you add the following comment here for 041 and 042.

```
LANG:JP このチュートリアルや UI は Token-2022 を使った NFT を作ります。Token を使った古い NFT も念の為探索します。
LANG:EN This tutorial and UI create NFTs using Token-2022. We will also explore older NFTs created with the traditional Token standard, just in case.
```

I think it is nice that we explain why devs need to search NFTs with both Token-2022 and Token.